tests/test_auxtel_close_dropout_door.py

```diff
@@ -0,0 +1,56 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+
+from lsst.ts import standardscripts
+from lsst.ts.observatory.control.mock import ATCSMock
+from lsst.ts.standardscripts.auxtel.atdome import CloseDropoutDoor
+
+
+class TestCloseDropoutDoor(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    async def basic_make_script(self, index):
+        self.script = CloseDropoutDoor(index=index)
+        self.atcs_mock = ATCSMock()
```

Comment:
It is better not to use `ATCSMock`, they are to be deprecated, use mocks instead.